### PR TITLE
Make the class more robust

### PR DESCRIPTION
**Pull request by phobson**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,6 @@ matrix:
     - python: 2.7
       env:
         - COVERAGE=false
-    - python: 3.3
```
Comment (tacaswell) [inline .travis.yml]: my favorite way of making tests pass :stuck_out_tongue_winking_eye:

**Comment by tacaswell** [comment]
Dropping 3.3 is probably a good call as we have dropped support for it upstream as well.

**Comment by phobson** [comment]
@tacaswell while I've got your attention[1], can you look at `probscale.py` and check for any horrific omissions or errors. I *really* want this in upstream MPL, and I've removed the scipy dependency, but (to my mind) there's enough magic the scales that I'm not terribly comfortable with what I've done. Also, a definitive "no -- this won't be in upstream MPL" would be very much understood.

[1] I know we're all busy. So feel free to ignore the request.

**Comment by tacaswell** [comment]
I have not given it read over yet (but will put it someplace in my semi-infinite to-do list)

It seems a bit niche (but maybe I just live in a niche that does not use these) and there is a general feeling to start pushing domain specific stuff _out_ of mpl core.  What can definitively be done is move this repo into the mpl org.

**Comment by phobson** [comment]
re: move domain-specific things out of mpl-core. 
I hear that. I think pure-python `mpl-scales` repo with logit, probability, and other scales would be awesome. 


I also just remembered that I was a little cavalier/gung-ho about adding you as a collaborator. If this repo is too noisey/distracting and you want me to undo that, let me know.

**Comment by tacaswell** [comment]
Sorry for being non-responsive.

I watch numpy, pandas, scipy, and mpl on gh, this project is not noisy at all.

re `mpl-scales`, yes, there as general support for that idea (and unifying how we do axis scales and `Norm`), just a lack of resource to do it.

**Pull request merged**